src/components/ContentPreview/Header.js

```diff
@@ -30,6 +30,11 @@ type Props = {
     token: ?string,
 };
 
+const LoadableContentOpenWith = AsyncLoad({
+    loader: () =>
+        import(/* webpackMode: "lazy", webpackChunkName: "content-open-with" */ '../ContentOpenWith/ContentOpenWith'),
```

Comment:
Thanks for adding this, was this being loaded by webapp even though it will never be used?

Reply:
Yes everything is loaded by webapp which is in ContentPreview. We are also loading every API even though we are only using a couple

Reply:
ContentOpenWith has an index.js, is ContentOpenWith/ContentOpenWith needed?